### ISSUE TITLE
Image list not updated after saving

### ISSUE THREAD

**Issue by TobsBobs** [joomla-projects/media-manager-improvement#576]
### Steps to reproduce the issue
Edit an image
Save the image
Go to the image list
The old image is displayed

### Expected result
The image list should be refreshed and the edited image is displayed.

### Actual result
The image list is not refreshed and the old image is displayed.

### System information (as much as possible)
php: Windows NT TOBSBOBS 10.0 build 17134 (Windows 10) i586
dbserver: mysql
dbversion: 5.5.5-10.1.33-MariaDB
dbcollation: latin1_swedish_ci
dbconnectioncollation: utf8_general_ci
phpversion: 7.2.6
server: Apache/2.4.33 (Win32) OpenSSL/1.1.0h PHP/7.2.6
sapi_name: apache2handler
version: Joomla! 4.0.0-alpha5-dev Development [ Amani ] 22-July-2018 22:58 GMT
useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:62.0) Gecko/20100101 Firefox/62.0

### Additional comments
![fehler refresh](https://user-images.githubusercontent.com/816779/43991768-e564a9ac-9d73-11e8-9664-ef2d0a0b8b2c.gif)

**This error happens in Firefox, Chrome and Edge.**

**Comment by schnuti** [comment]
I can confirm this issue. I have almost the same configuration as toby.
@TobsBobs This is without the save fix? Could you open the Chrome-developer->Console. Any messages when saving? I get some about slow system but the changed image is then reloaded.

@laoneo Do you know if something was changed for image loading. As far as I remember it used to work except when the image wasn't saved correctly with Firefox.  I read something about streaming...

**Comment by schnuti** [comment]
It seems to be a problem where the browser cache is involved.
With Chrome:
- The image was changed on disk. (copied the original back)
- Reloaded page Did Not get the image from disk. Kept the one in the cache.
- Opened the Chrome-Developer
- Reloaded page got the original from disk.
i.e. hard to test to get the problem.

**Comment by TobsBobs** [comment]
@schnuti 
Yes, this was without your save fix.
In Chrome-developer-Console is no error message visible. I think you are right. 
The first thing we should fix is the save button.

I'll play with it the next few days.

**Comment by schnuti** [comment]
I'm out of ideas. It seems to be completely random.
Here some docs from the same session (in FF). Did not leave the MM views.
1. Original load of a directory. The images are requested at the bottom of the list. You can see 2 images loaded.
![requests-in-ffdeveloper](https://user-images.githubusercontent.com/956537/44000656-6fb8b804-9e24-11e8-8927-4e8d0535b94a.jpg)

2. Load after saving image. This one worked. One image reloaded other from cache. 
![requests-in-ffdeveloper-after-save](https://user-images.githubusercontent.com/956537/44000685-f0ee6e96-9e24-11e8-976d-240cb36793e4.jpg)

3. Load after saving image. **This one did not work.** No image at the bottom of the request list.
![requests-in-ffdeveloper-not-reloaded](https://user-images.githubusercontent.com/956537/44000715-5286ea0c-9e25-11e8-9b44-37ef840e5a19.jpg)

**Comment by TobsBobs** [comment]
Thanks @schnuti !

Any updates after two month with no response?

**Comment by laoneo** [comment]
It needs some browser cache cleanup after save.

**Comment by TobsBobs** [comment]
@laoneo yes that's right. But with this issues i can not properly test the new media manager and joomla 4 is far away from beta status.

I will wait and see what happens. Thanks!
